mssql-ballerina/tests/connection-init-test.bal

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/sql;
+import ballerina/test;
+
+string connectDB = "CONNECT_DB";
+
+@test:Config {
+    groups: ["connection", "connection-init"]
+}
+function testConnectionWithNoFields() {
+    Client|sql:Error dbClient = new ();
+    test:assertTrue(dbClient is sql:Error, "Initialising connection with no fields fails.");
+}
+
+@test:Config {
+    groups: ["connection", "connection-init"]
+}
+function testWithURLParams() {
+    Client dbClient = checkpanic new (host=host, port = port, user = user, password = password, database = connectDB);
```

Comment:
Don't use `checkpanic`. instead, use `check` and returns the error

Reply:
check all the places

Reply:
Fixed